web/src/components/Feedback.tsx

```diff
@@ -24,15 +24,14 @@ export const Container = styled.div<{ fullWidth?: boolean }>`
   font-size: ${props => props.theme.fonts.contentFontSize};
   overflow: auto;
   align-self: center;
-  gap: ${props => (props.fullWidth ? '5px' : 0)};
+  gap: ${props => (props.fullWidth ? '7px' : 0)};
```

Comment:
The problem is now that since the padding was removed the gap is missing for not fullWidth which is used for the feedback within a modal.
Go to any page and check the left toolbar for Feedback and open it. 
You now also need this gap there so remove that `fullWidth` variable and condition and just set the `7px`